runtime-transaction/src/lib.rs

```diff
@@ -1,10 +1,6 @@
 #![cfg_attr(feature = "frozen-abi", feature(min_specialization))]
```

Comment:
@tao-stones we should remove this here as well. There's no need for any of the stuff in this crate to be ABI stable.

Reply:
Good point, can remove it in this PR, otherwise I have https://github.com/anza-xyz/agave/pull/4016 for it